client/state/action-types.js

```diff
@@ -543,7 +543,10 @@ export const MAILCHIMP_SETTINGS_LIST = 'MAILCHIMP_SETTINGS_LIST';
 export const MAILCHIMP_SETTINGS_UPDATE = 'MAILCHIMP_SETTINGS_UPDATE';
 export const MAILCHIMP_SETTINGS_UPDATE_SUCCESS = 'MAILCHIMP_SETTINGS_UPDATE_SUCCESS';
 export const MAILCHIMP_SETTINGS_UPDATE_FAILURE = 'MAILCHIMP_SETTINGS_UPDATE_FAILURE';
-
+export const MY_SITES_SIDEBAR_SITE_TOGGLE = 'MY_SITES_SIDEBAR_SITE_TOGGLE';
```

Comment:
do we really need one action per menu item?

Reply:
though i guess the alternative would be MY_SITES_SIDEBAR_TOGGLE and pass along a const or symbol for each thing in the sidebar... so maybe it's not really much of a win to consolidate